recipes/cabal/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cabal" %}
+{% set version = "2.4.1.0" %}
+{% set sha256 = "abe4838dc0ef2a25e482d02480cd89eda3f8ccf6d8c4edf2063ae5723d8f12b5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/haskell/cabal/archive/Cabal-v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ghc
+    - zlib
```

Comment:
zlib can be removed from the build section and ghc and gmp probably as well

Reply:
GHC is needed for building, zlib is needed for linking in zlib.h, as well as libgmp.10.so

Reply:
But it should be enough in the `host` section.

Reply:
Ah, ok :-) I will change it!

Reply:
Ok, removed the compiler flags